mozilla/activity-stream
# Bug 1592724 - campaign to flight value change

To test:
1. You need to enable DS and spocs.
2. Ensure you see spocs.
3. Ensure spocs frequency cap out.
4. Ensure spocs can be blocked.
5. Change `browser.newtabpage.activity-stream.discoverystream.spocs-endpoint` to `https://getpocket.cdn.mozilla.net/v3/firefox/unique-spocs`
6. Ensure steps 2-4 work still.
